components/tech-docs/queue-proxy-docs/mkdocs.yaml

```diff
@@ -0,0 +1,8 @@
+site_name: 'Queue Proxy'
```

Comment:
I'm not sure if it'll work with this file nested, it might need to go to the root dir.

Reply:
Nevermind, I stand corrected.